src/apps/answers/service.py

```diff
@@ -284,6 +284,9 @@ async def _get_answer_relation(
 
             return Relation.other
 
+        if is_take_now_relation(relation) and is_valid_take_now_relation(relation):
+            return Relation.other
+
         return relation.relation
```

Comment:
We should not use this temporary relation to determine relation between subject to store it in answer table, this temporary relation is only to give access to subject data

Maybe we should have a call about it when possible. cc: @sultanofcardio